IBX-2752 Raptor CDP connector (#1756)
ezsystems/developer-documentation: docs/cdp/cdp.md

> # Customer Data Platform (CDP)

Author: ```suggestion
---
description: Ibexa CDP a software system designed to collect and organize customer data from multiple sources to build comprehensive customer profiles.
---

# Customer Data Platform (CDP)
```

> # Customer Data Platform (CDP) [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]

Reviewer: Not sure if I understand the badge correctly - if it means CDP is available from Experience edition then this is wrong as it's a standalone package that can be installed alongside every product edition (Content, Experience, Commerce). Maybe we should add Content badge as well.

Reviewer 2: So let's remove them.

## What is Ibexa CDP

Ibexa CDP helps you solve one of the hardest challenges facing business world today:
building unique experiences for your customers.

> With Ibexa CDP you will be able to track and aggregate data on your customers' activity on multiple channels.

Reviewer 2: ```suggestion
With Ibexa CDP you will be able to track and aggregate data of your customers' activity on multiple channels.
```

It will allow you to create individual customer profiles that enable you to personalize their experience on your platform.

> ![](img/cdp_control_panel.png)

Author: ```suggestion
![Ibexa CDP control panel](img/cdp_control_panel.png)
```

## How it works

Ibexa CDP unifies customer data across your organization
to help you activate your users and provide them with real-time engagement.
With defined audiences you can target your user segments at the right time,
through the most used channel, with the relevant message, content, or products.

The customer data are collected through the system of trackers embedded in different parts of your page.

> For more information on activation and trackers, see[CDP activation documentation](cdp_activation.md).

Reviewer: ```suggestion
For more information on activation and trackers, see [CDP activation documentation](cdp_activation.md).
```

> For more information on activation and trackers go to [CDP activation documentation](cdp_activation.md).

Reviewer 2: ```suggestion
For more information on activation and trackers, see[CDP activation documentation](cdp_activation.md).
```
